openapi/components/schemas/Common/CommonBankAccount.yaml

```
@@ -43,7 +43,13 @@ properties:
     allOf:
       - $ref: ../Contact/ContactObject.yaml
   fingerprint:
-    description: Unique value which identifies the bank account. This value contains alphanumeric characters.
+    description: |-
+      Unique value which identifies the bank account.
+      This value contains alphanumeric characters.
+      Depending on the type of bank account number,
+      a bank account fingerprint is generated using one of the following:
+      - BBAN: `last4` and `routingNumber` values.
+      - IBAN: First 8 digits of the IBAN and the `last4` value.
```

```suggestion
      - IBAN: First 8 characters of the IBAN and the `last4` value.
```
IBAN contains alphanumeric characters, i.e. DE75512108001245126199